introduction.md

```diff
@@ -1 +1,13 @@
 Hello, I am levi stanley.  I like this code school:  http://www.codeschool.com.  Hmm they were talking about a language JavaScript.
+
+* PERL
+* PHP
+* ADA
+* Vax Assembly
+* Assembly x86
+* Ruby
+* Java
+* C#
+* C
+* C++
+* 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?